src/vic3/data/map.rs

```diff
@@ -45,3 +45,134 @@ impl DbKind for MapLayer {
         vd.field_value("visibility_tags");
     }
 }
+
+#[derive(Clone, Debug)]
+pub struct MapMode {}
+
+inventory::submit! {
+    ItemLoader::Normal(GameFlags::Vic3, Item::MapMode, MapMode::add)
+}
+
+// LAST UPDATED VIC3 VERSION 1.5.13
```

Comment:
Best mark where these are from: `gfx/map/map_modes/map_modes.txt`

Reply:
Turns out that gradient border settings have a .txt file defining them :) I'll add a loader for that too